pypowsybl/report.py

```diff
@@ -16,3 +16,7 @@ def __repr__(self) -> str:
     @property
     def _reporter_model(self) -> _pp.JavaHandle:
         return self._reporter_model_handle
+
+    @property
+    def json(self) -> str:
```

Comment:
I would slightly prefer to name it `to_json()` and make it a method, to make it explicit there is some "work" under the hood (also, if we need it later, this will allow for extra arguments such as pretty printing etc).
This will look familiar to pandas users where there are methods suchs as `to_csv()`, ...

Reply:
done: changed to to_json()